cmd/generate/generate.go

```diff
@@ -326,6 +331,43 @@ func main() {
 		if err := os.WriteFile(filepath.Join(output, "images.yaml"), mapping, fs.ModePerm); err != nil {
 			log.Fatal("Write images mapping file ", err)
 		}
+	} else if generators == GenerateMustGatherDockerfileOption {
+		if templateName != mustGatherDockerfileTemplateName {
+			log.Fatal("Unknown template name: " + templateName)
+		}
+		metadata, err := project.ReadMetadataFile(projectFilePath)
+		if err != nil {
+			if !errors.Is(err, os.ErrNotExist) {
+				log.Fatal("Failed to read project metadata file: ", err)
+			}
+			log.Println("File ", projectFilePath, " not found")
+			metadata = nil
+		}
+
+		projectName := mustGatherDockerfileTemplateName
+		var projectDashCaseWithSep string
+		if projectName != "" {
+			projectDashCaseWithSep = projectName + "-"
+		}
```

Comment:
```suggestion
		projectDashCaseWithSep := projectName + "-"
```
I don't think we need this `if`, as `projectName` will not be empty here